core/src/main/java/io/opencensus/tags/Tag.java

```diff
@@ -142,8 +142,7 @@ public static TagString create(TagKeyString key, TagValueString value) {
      * @param value the tag value.
      * @return a {@code TagLong} with the given key and value.
      */
-    // TODO(sebright): Make this public once we support types other than String.
-    static TagLong create(TagKeyLong key, long value) {
+    public static TagLong create(TagKeyLong key, long value) {
```

Comment:
I realized that there is no reason to hide any of the Tag factory methods, because we already hide the TagKey factory methods for the unsupported types, which means that it still isn't possible to create a `TagLong` or `TagBoolean`.

Since our documentation already explains that tags can have different types, I found that it was easier to expose these methods so that the documentation can reference them.  Another reason to expose all Tag factory methods is that they are needed by the `tags` implementation, which is in a different package.